LevelEditor/SliderWidget.py

```diff
@@ -0,0 +1,182 @@
+from PySide.QtCore import *
+from PySide.QtGui import *
+from collections import OrderedDict
+
+sliders = None
+
+
+class AdjustValues(QWidget):
+    valuesChanged = Signal(list)
+
+    def __init__(self, val_list, min_val, max_val):
+        super(AdjustValues, self).__init__()
+        self.val_list = val_list
```

Comment:
Do you actually need to store the val_list? Doesn't look like it gets used outside of `__init__`, and storing fewer things is better than more.